homeassistant/components/calendar/__init__.py

```diff
@@ -250,6 +257,20 @@ def _validate_rrule(value: Any) -> str:
     extra=vol.ALLOW_EXTRA,
 )
 
+SERVICE_LIST_EVENTS: Final = "list_events"
+SERVICE_LIST_EVENTS_SCHEMA: Final = vol.All(
+    cv.has_at_most_one_key(EVENT_END_DATETIME, EVENT_DURATION),
+    cv.make_entity_service_schema(
+        {
+            vol.Optional(EVENT_START_DATETIME): datetime.datetime,
```

Comment:
Start and end time should be inclusive.

Reply:
Start is now made optional, defaulting to now. Should it still be inclusive?  I understand /what/ your comment is saying, but not sure /why/ so want to make sure I understand the point you're making.

Reply:
Inclusive means optional. If either of start or end is set, both should be set. I think we should only default start to now when using duration.

Reply:
I see, i made it optional on purpose since it seemed unnecessarily rigid to make that distinction. I get why it makes sense conceptually to have start and end paired, but in practice it seems nice to not have to make users care about it if they want to just use a duration or switch to an end time later.

Do you have much conviction about it? I can adjust if you do, but just wanted to check one more time.

Reply:
Ok, we can drop the inclusive.